### DISCUSSION TITLE
About that tiny Latex editor

### DISCUSSION THREAD

**Discussion by Soothfast** [xournalpp/xournalpp#5559] [Q&A]
Last I checked, over a year ago when I was using xournalpp on a Windows machine, the only way to get a Latex editing window that was larger than two postage stamps was to use beta versions of the program (something about "nightly updates").  So is that it?  There is no way to get a decent-sized text field to type in on v1.2.3?  I find this to be the second most problematic thing about xournalpp, at least for Linux.  It is really crippling, but I'm not enthused about using potentially buggy beta versions of the program.

All I can find under preferences are half a dozen different color schemes for the Latex editor, but the size of the text field remains stubbornly un-resizable for some reason.

**Comment by rolandlo** [comment]
In v1.2.3 you can simply resize the LaTeX editor window by dragging one of the edges. You can also drag the line between the text field where you edit your formula and the preview above it. That should be pretty intuitive.

**Reply by Soothfast** [reply]
Did that also work in v1.2.2?   Because I tried those things you suggest with that version, and couldn't get anything to budge.  

I applied a surgeon's hand this time, with screen magnified, in v1.2.3, and found the sweet spots.  Okay, you're right.  It's a few pixels wide, but there is a place to snag the edge of the window and resize it.  Thanks.
